src/ts/directives/calendar.ts

```diff
@@ -26,6 +28,22 @@ export let calendarComponent = ng.directive('calendar', function () {
                 $scope.display.editItem = false;
                 $scope.display.createItem = false;
 
+                let calendar_options = {showQuarterHours:false};
+                $scope.toogleQuarterHours = () => {
+                    if(!Me.preferences.calendar_options){
+                        Me.preferences.calendar_options = {};
+                    }
+                    Me.preferences.calendar_options["showQuarterHours"] = $scope.display.showQuarterHours;
```

Comment:
on peut sûrement accéder à la propriété avec la notation:
```suggestion
                    Me.preferences.calendar_options.showQuarterHours = $scope.display.showQuarterHours;
```